### PR TITLE
*/*: Port to EAPI 7

### PR DESCRIPTION
**Pull request by jsmolic**

### CODE REVIEW

File: media-sound/dvda-author/dvda-author-10.06-r1.ebuild [hunk 1]
```diff
@@ -36,16 +40,3 @@ src_configure() {
...
-	#                   Add '+' to parent make rule.
-	emake -j1
-}
-
-src_install() {
-	newbin src/dvda ${PN}
-	insinto /etc
-	doins ${PN}.conf
-	dodoc AUTHORS BUGS ChangeLog EXAMPLES HOWTO.conf LIMITATIONS NEWS TODO
-}
```
Comment (SoapGentoo) [inline media-sound/dvda-author/dvda-author-10.06-r1.ebuild:51]: `src_install` works just fine with `default_src_prepare`?

File: net-misc/astmanproxy/astmanproxy-1.28.2.ebuild [hunk 2]
```diff
@@ -1,25 +1,25 @@
 # Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
+EAPI=7
 
-inherit base flag-o-matic multilib toolchain-funcs
+inherit flag-o-matic multilib toolchain-funcs
```
Comment (SoapGentoo) [inline net-misc/astmanproxy/astmanproxy-1.28.2.ebuild:6]: `multilib` can go (`get_libdir` is part of EAPI 6+)

File: net-misc/astmanproxy/astmanproxy-1.28.2.ebuild [hunk 3]
```diff
@@ -1,25 +1,25 @@
...
 
 DESCRIPTION="Proxy for the Asterisk manager interface"
 HOMEPAGE="https://github.com/davies147/astmanproxy/"
 SRC_URI="https://github.com/davies147/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
 
-IUSE=""
-
 SLOT="0"
 LICENSE="GPL-2"
 KEYWORDS="~amd64 ~ppc ~x86"
```
Comment (SoapGentoo) [inline net-misc/astmanproxy/astmanproxy-1.28.2.ebuild:14]: sort this block

File: net-misc/astmanproxy/astmanproxy-1.28.2.ebuild [hunk 4]
```diff
@@ -1,25 +1,25 @@
...
+	"${FILESDIR}"/${P}-gentoo.diff
+	"${FILESDIR}"/${P}-fno-common.patch
+)
 
 src_prepare() {
-	base_src_prepare
+	default
 
 	# Fix multilib
 	sed -i -e "s#/usr/lib/#/usr/$(get_libdir)/#" "${S}/Makefile" \
```
Comment (SoapGentoo) [inline net-misc/astmanproxy/astmanproxy-1.28.2.ebuild:25]: `"${S}/Makefile"` -> `Makefile`

File: net-misc/astmanproxy/astmanproxy-1.28.2.ebuild [hunk 5]
```diff
@@ -36,14 +36,11 @@ src_compile() {
 src_install() {
 	dosbin astmanproxy
 
-	dodoc README VERSIONS
-
-	docinto samples
-	dodoc samples/*
+	dodoc -r samples
+	einstalldocs
```
Comment (SoapGentoo) [inline net-misc/astmanproxy/astmanproxy-1.28.2.ebuild:40]: I assume `VERSIONS` is just a dud?

File: www-client/uget/uget-2.2.1.ebuild [hunk 6]
```diff
@@ -28,13 +28,14 @@ RDEPEND="
 	appindicator? ( dev-libs/libappindicator:3 )
 	gstreamer? ( media-libs/gstreamer:1.0 )
 	libnotify? ( x11-libs/libnotify )
-	openssl? ( dev-libs/openssl:0 )
-	"
-DEPEND="${RDEPEND}
+	openssl? ( dev-libs/openssl:0 )"
```
Comment (SoapGentoo) [inline www-client/uget/uget-2.2.1.ebuild:31]: check if this links, and if it does, add `:0=`

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

File: media-sound/dvda-author/dvda-author-10.06-r1.ebuild (hunk 1 above)
Comment (jsmolic) [reply media-sound/dvda-author/dvda-author-10.06-r1.ebuild:51]: Yep, we do have a bit more files installed with default src_install

File: net-misc/astmanproxy/astmanproxy-1.28.2.ebuild (hunk 2 above)
Comment (jsmolic) [reply net-misc/astmanproxy/astmanproxy-1.28.2.ebuild]: Ah, that's right, dropped

File: net-misc/astmanproxy/astmanproxy-1.28.2.ebuild (hunk 3 above)
Comment (jsmolic) [reply net-misc/astmanproxy/astmanproxy-1.28.2.ebuild:14]: Done

File: net-misc/astmanproxy/astmanproxy-1.28.2.ebuild (hunk 4 above)
Comment (jsmolic) [reply net-misc/astmanproxy/astmanproxy-1.28.2.ebuild]: Done

File: net-misc/astmanproxy/astmanproxy-1.28.2.ebuild (hunk 5 above)
Comment (jsmolic) [reply net-misc/astmanproxy/astmanproxy-1.28.2.ebuild]: It's something like changelog, doesn't get picked up with `einstalldocs`. Changed it to previous version

File: www-client/uget/uget-2.2.1.ebuild (hunk 6 above)
Comment (jsmolic) [reply www-client/uget/uget-2.2.1.ebuild:31]: Does not link against openssl

File: media-sound/dvda-author/dvda-author-10.06-r1.ebuild [hunk 7]
```diff
@@ -36,16 +40,3 @@ src_configure() {
 		--with-config=/etc \
```
Comment (SoapGentoo) [inline media-sound/dvda-author/dvda-author-10.06-r1.ebuild:40]: EPREFIX

File: www-client/uget/uget-2.2.1.ebuild [hunk 8]
```diff
@@ -16,9 +16,9 @@ IUSE="aria2 appindicator control-socket +gnutls gstreamer libnotify nls openssl
 REQUIRED_USE="^^ ( gnutls openssl )"
 
 RDEPEND="
-	>=net-misc/curl-7.19.1
+	dev-libs/glib
```
Comment (SoapGentoo) [inline www-client/uget/uget-2.2.1.ebuild:19]: keep the `:2` part

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

File: media-sound/dvda-author/dvda-author-10.06-r1.ebuild (hunk 7 above)
Comment (jsmolic) [reply media-sound/dvda-author/dvda-author-10.06-r1.ebuild]: Thanks, added

File: www-client/uget/uget-2.2.1.ebuild (hunk 8 above)
Comment (jsmolic) [reply www-client/uget/uget-2.2.1.ebuild]: Done

**Review by SoapGentoo — APPROVED** [review]

**Pull request closed without merging**